src/AbstractWalkers.jl

```diff
@@ -0,0 +1,147 @@
+"""
+Module containing abstract definitions for walkers.
+"""
+module AbstractWalkers
+
+using AtomsBase
+using Unitful
+using ..Potentials
+using ..EnergyEval
+
+export AtomWalker, AtomWalkers, LJAtomWalkers
```

Comment:
@yangmr04, can you explain what `export` does here?

Reply:
`export` export the functions that can be called directly. For example, here one can do `using FreeBird; AtomWalkers(args...)`, otherwise the method can only be called such as `FreeBird.AbstractWalkers.AtomWalkers(args...)`, which is often referred as a "internal function".